frontend_tests/puppeteer_tests/settings.ts

```diff
@@ -184,7 +184,11 @@ async function test_edit_bot_form(page: Page): Promise<void> {
     await common.wait_for_micromodal_to_close(page);
 }
 
+// Disabled the below test due to non-deterministic failures.
+// The test often fails to close the modal.
+// TODO: Debug this and re-enable with a fix.
 async function test_invalid_edit_bot_form(page: Page): Promise<void> {
+    return;
     const bot1_email = "1-bot@zulip.testserver";
```

Comment:
This is a commit in `main`; the PR author appears to have accidentally included that commit in their PR.  `git fetch; git rebase upstream/main` should drop it.

Reply:
Oh! I see, Shubh did mention it in the previous commit (I probably missed it):

> ... was added because of resolving merge conflicts with main branch.